### PR TITLE
fixed routes

### PR DESCRIPTION
**Pull request by djkowa**
In about page when flat button 'About' is pressed it closes the drawer.

### CODE REVIEW

File: lib/router/router.dart [hunk 1]
```diff
@@ -3,17 +3,19 @@ import 'package:flutter/widgets.dart';
...
-
-
 class Router {
   static Route generator(RouteSettings settings) {
     switch (settings.name) {
       case RouteName.homePage:
-        return MaterialPageRoute(builder: (_) => HomePage());
+        return MaterialPageRoute(
+          builder: (_) => HomePage(),
+          settings: RouteSettings(name: '/'),
```
Comment (matejthetree) [inline lib/router/router.dart]: sta bez ovog nema imena?
super find.

Jedna zamjena, umjesto da radite novi route settings, primite settings gore u switch

`settings: settings`

File: lib/ui_pages/about_page.dart [hunk 2]
```diff
@@ -8,6 +9,11 @@ class AboutPage extends StatelessWidget {
     return Scaffold(
       endDrawer: CustomDrawer(),
       appBar: AppBar(
+        leading: IconButton(
+          icon: Icon(Icons.arrow_back),
+          onPressed: () =>
+              Navigator.pushReplacementNamed(context, RouteName.homePage),
```
Comment (matejthetree) [inline lib/ui_pages/about_page.dart]: ovdje neznate hoce li doci u about iz home pagea.

Defaultni leading je bio dobar. samo <- da napravi pop.
To treba ostati isto

File: lib/widgets/drawer.dart [hunk 3]
```diff
@@ -18,11 +20,16 @@ class CustomDrawer extends StatelessWidget {
...
               child: Text(
                 'About',
-                    style: TextStyle(fontSize: 20,
-                    fontStyle: FontStyle.italic),
+                style: TextStyle(fontSize: 20, fontStyle: FontStyle.italic),
               ),
-              onPressed: () =>
-                  Navigator.pushNamed(context, RouteName.aboutPage),
+              onPressed: () {
+                route!= '/aboutPage'?
```
Comment (matejthetree) [inline lib/widgets/drawer.dart]: super.

Umjesto '/aboutPage' koristite `RouteName.aboutPage`

File: lib/widgets/drawer.dart [hunk 4]
```diff
@@ -18,11 +20,16 @@ class CustomDrawer extends StatelessWidget {
...
+                Navigator.pushNamed(context, RouteName.aboutPage):
+              Navigator.pop(context);
```
Comment (matejthetree) [inline lib/widgets/drawer.dart]: cini mi se da nije formatirano dobro

**Review by matejthetree — CHANGES_REQUESTED** [review]

File: lib/router/router.dart (hunk 1 above)
Comment (djkowa) [reply lib/router/router.dart]: resolved

**Comment by djkowa** [comment]
we have applied changes to the routes as requested

**Review by matejthetree — APPROVED** [review]

**Pull request merged**